### PR TITLE
Add source citations to bottom of project input page

### PR DESCRIPTION
**Pull request by CNimmo16**
Add citations list - issue #9. Note that list is hard coded rather than using data from google sheets, not sure if this is the best solution or is more versatility needed?

### CODE REVIEW

**Comment by pjconrad** [comment]
I don’t know if more versatility is needed. We may add new references over time, but likely to be few and not very often.

I suppose they could be copied into the google sheets and referenced that way, but defer to Edan.

**Comment by edanweis** [comment]
Thanks Cameron

1. Change the target attr of the href anchor to `_blank`
2. Change the text to short and snappy? "Visit climatepositivedesign.com/resources to learn more" @pjconrad what do you think?

File: src/components/application/CalculatorSheet.vue [hunk 1]
```diff
@@ -72,6 +72,24 @@
...
+    <p>Gu, C., Crane, J., Hornberger, G., & Carrico, A. (2015). The effects of household management practices on the global warming potential of urban lawns. Journal of environmental management, 151, 233-242.</p>
+    <p>McPherson, EG.; Simpson, JR. (1999). Carbon dioxide reduction through urban forestry: Guidelines for professional and volunteer tree planters. Gen. Tech. Rep. PSWGTR-171. Albany, CA: Pacific Southwest Research Station, Forest Service, U.S. Department of Agriculture.</p>
+  </div>
+  <div v-if="type === 'cost'">
+    <p>Banks, J. L., & McConnell, R. (2015, April). National Emissions from Lawn and Garden Equipment. In International Emissions Inventory Conference, San Diego.</p>
+    <p>Gu, C., Crane, J., Hornberger, G., & Carrico, A. (2015). The effects of household management practices on the global warming potential of urban lawns. Journal of environmental management, 151, 233-242.</p>
+    <p>Lal, R. (2004). Carbon emissions from farm operations. Environ. Int. 30 (7), 981-990.</p>
+  </div>
+  <p class="note">For a full list of references, refer to the <a href="https://climatepositivedesign.com/resources/">Resources page</a> of website.</p>
```
Comment (edanweis) [inline src/components/application/CalculatorSheet.vue]: Add `target="_blank"` to link

**Comment by CNimmo16** [comment]
Updated with requested - added text change in separate commit so it can be kept in or reverted to original text.

**Pull request merged**